pkg/rbac/sharedData.go

```diff
@@ -51,16 +55,86 @@ var managedClusterResourceGvr = schema.GroupVersionResource{
 	Resource: "managedclusters",
 }
 
-func (cache *Cache) PopulateSharedCache(ctx context.Context) error {
+func (shared *SharedData) GetPropertyTypes() (map[string]string, error) {
```

Comment:
I think this can be private.
```suggestion
func (shared *SharedData) getPropertyTypes() (map[string]string, error) {
```

Reply:
Yes, meant to change that back to private.